src/Geta.Optimizely.Sitemaps/Pages/Geta.Optimizely.Sitemaps/Index.cshtml

```diff
@@ -136,14 +137,15 @@
                 <form method="post">
                     <tr class="insert">
                         <td class="sitemap-name">
-                            @if (Model.ShowHostsLabel)
-                            {
-                                <label asp-for="SitemapViewModel.RelativePath"></label>
-                            }
                             @if (Model.ShowHostsDropDown)
                             {
                                 <select asp-for="SitemapViewModel.SiteUrl" asp-items="Model.SiteHosts"></select>
                             }
+                            else
+                            {
+                                <label asp-for="SitemapViewModel.RelativePath"></label>
+                                <input type="hidden" class="form-control align-middle" asp-for="SitemapViewModel.SiteUrl" value="@Model.HostLabel"/>
```

Comment:
Do you need CSS class for the hidden input?

Reply:
Is `asp-for="SitemapViewModel.SiteUrl"` correct here? Shouldn't it be `asp-for="SitemapViewModel.RelativePath"`?

Reply:
It is correct, the issue only existed if the host dropdown wasn't shown - when there's only 1 host.